src/StreamJsonRpc.Shared/Reflection/MethodNameAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft;
+
+namespace StreamJsonRpc
+{
+    /// <summary>
+    /// Attribute which allows a method to have an alternate name.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
```

Comment:
Allow multiple = true would allow the same method to have different aliases? Seems to be useful.